workspaces/optic-engine-wasm/src/lib.rs

```diff
@@ -380,6 +380,28 @@ pub fn spec_resolve_response(
     .map_err(|err| JsValue::from(format!("responses could not be serialized: {:?}", err)))
 }
 
+#[wasm_bindgen]
+pub fn spec_endpoint_delete_commands(
+  spec: &WasmSpecProjection,
+  path_id: String,
+  method: String,
+) -> Result<String, JsValue> {
+  let endpoint_queries = spec.endpoint_queries();
+
```

Comment:
Why are we using endpoint queries here and aren't putting this logic anywhere else? Lets look at the typical CQRS cycle:

`... -> Events -> Projections -> [Queries] -> Commands -> Events -> ...`.

Since we're generating commands here, that makes the obvious place to implement these the queries (drawn as optional above, as sometimes projections are simple enough that we implement a query straight on it, but conceptually it's pretty much the same thing).

So why _Endpoint_Queries, specifically?

So far, the Events, Projections and Queries have all been broken up in `Endpoint*`, `Shape*` and `Rfc*`, describing request level, shape / body level and more metadata like parts of the chain, respectively. They're all variant of grouping `Spec*` variants. There are _other_ projections that cross some of these concerns, but generally it's a partitioning that has held up really well.